airflow/providers/google/suite/hooks/sheets.py

```diff
@@ -48,13 +46,11 @@ class GSheetsHook(CloudBaseHook):
 
     def __init__(
         self,
-        spreadsheet_id: str,
         gcp_conn_id: str = 'google_cloud_default',
```

Comment:
Has this Hook been released in any version of Airflow? If so, then we should maintain backward compatibility. You can do it with a decorator. We have the similar decorator in the codebase: https://github.com/apache/airflow/blob/master/airflow/providers/google/cloud/hooks/dataflow.py#L52-L81

Reply:
I don't see this hook here:
https://github.com/apache/airflow/tree/v1-10-stable/airflow/contrib/hooks